proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -87,12 +108,11 @@ message GetModuleDocumentationResponse {
 // ModuleDocumentation provides the name of the module and associated documentation.
 message ModuleDocumentation {
   string name = 1;
-  repeated string packages = 2;
```

Comment:
Are we sure we want to delete this, or do we just want to add a new method?

Reply:
we want to remove it from the method, so that even when we cannot build the image and get the packages list, we can still provide the documentation